src/api/core/index.ts

```diff
@@ -11,4 +11,20 @@ baseRequest.interceptors.request.use(
   (error) => Promise.reject(error),
 );
 
+const handleRequest = (config: AxiosRequestConfig) => config;
+
+const createApiMethod =
+  (axiosInstance: AxiosInstance, methodType: Method) => (config: AxiosRequestConfig) => {
+    return axiosInstance({ ...handleRequest(config), method: methodType });
+  };
```

Comment:
proposal;

```suggestion
const createApiMethod =
  <T>(axiosInstance: AxiosInstance, methodType: Method) => (config: AxiosRequestConfig<T>) => {
    return axiosInstance({ ...handleRequest(config), method: methodType });
  };
```

여기도 위쪽 코멘트와 동일하네요!!